lib/ferto/client.rb

```diff
@@ -54,13 +54,24 @@ def initialize(opts = {})
     #   resource
     # @param aggr_id       [String] aggregation identifier
     # @param aggr_limit    [Integer] aggregation concurrency limit
+    # @param aggr_proxy    [String] the HTTP proxy to use for downloading the
+    #   resource, by default no proxy is used
+    # @param aggr_timeout  [Integer] the maximum time to wait for the resource
+    #   to be downloaded in seconds, by default it uses the timeout set in
+    #   the downloader's configuration
+    # @param aggr_useragent [String] the User-Agent string to use for
```

Comment:
Note: This depends on the downloader implementation, however since we have no PR for that yet I'll write it here.

I believe we shouldn't limit this feature to the "User-Agent" header - we should allow setting _any_ header the user wants. It might be a little bit more work to implement, but it's worth the effort. Otherwise, we'll have to change the code in downloader and ferto every time we want to set another header.

Also, is there a reason to tie this to the Aggregation, as opposed to the Job?